test/integration/provider/admin/backend_api_controller_test.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Provider::Admin::BackendApisControllerTest < ActionDispatch::IntegrationTest
```

Comment:
so the path of the file should be `test/integration/provider/admin/backend_apis_controller_test.rb` 😄 it is lacking the `s` in `apis` right now 😄 It seems a very small thing but it can lead to problems, and it is easy to fix now 😝(by `mv test/integration/provider/admin/backend_api_controller_test.rb test/integration/provider/admin/backend_apis_controller_test.rb`)